modules/cluster-management/src/integrationTest/java/org/apache/ignite/internal/cluster/management/topology/ItLogicalTopologyTest.java

```diff
@@ -172,7 +172,7 @@ void receivesLogicalTopologyEventsWithAttributes() throws Exception {
         assertThat(event.eventType, is(EventType.VALIDATED));
         assertThat(event.node.name(), is(secondIgnite.name()));
         assertThat(event.node.userAttributes(), is(NODE_ATTRIBUTES_MAP));
-        assertThat(event.node.storageProfiles(), containsInAnyOrder(STORAGE_PROFILES_LIST.toArray()));
+        assertThat(event.node.storageProfiles(), hasItems(STORAGE_PROFILES_LIST.toArray(new String[] {})));
```

Comment:
Because actually, despite of naming containsInAnyOrder check not for "contains", but for the full equals from the point of set.equals view.

Instead, here, we have some default profiles from the base test configs and I want to check that my custom profiles is in the list laos.